### PR TITLE
magento/magento2#25540: Products are not displaying infront end after updating product via importing CSV.

### PR DESCRIPTION
**Pull request by p-bystritsky**
### Fixed Issues (if relevant)

1. magento/magento2#25540: Products are not displaying infront end after updating product via importing CSV.

### Manual testing scenarios (*)

1. Create Simple product.
2. Create Bundle product, assign Simple to it.
3. Open Storefront, make sure Bundle product is visible and in stock.
4. Go to System -> Data Transfer -> Export and export products.
5. Go to System -> Data Transfer -> Import and import products.
6. Do reindex and flush cache.
7. Open Storefront, make sure Bundle product is visible and in stock.

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

**Comment by engcom-Echo** [comment]
@magenro run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Review by lenaorobei — COMMENTED** [review]
@p-bystritsky please address build failures. Thanks.

**Comment by p-bystritsky** [comment]
@lenaorobei done.

File: app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php [hunk 1]
```diff
@@ -6,26 +6,29 @@
...
- * Class Bundle
+ * Import entity Bundle product type.
  *
- * @package Magento\BundleImportExport\Model\Import\Product\Type
  * @SuppressWarnings(PHPMD.ExcessiveClassComplexity)
  */
 class Bundle extends \Magento\CatalogImportExport\Model\Import\Product\Type\AbstractType
 {
+    /**
+     * phpcs:disable Magento2.Commenting.ConstantsPHPDocFormatting
```
Comment (lenaorobei) [inline app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php]: What is the reason for this ignore annotation?

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Thanks for the update. Please clarify one review comment.

File: app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php (hunk 1 above)
Comment (p-bystritsky) [reply app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php]: Updated.

Comment (lenaorobei) [reply app/code/Magento/BundleImportExport/Model/Import/Product/Type/Bundle.php]: Thank you. I see now copy-paste detector is failing. Could you please check what is causing it? https://public-results-storage-prod.magento-testing-service.engineering/reports/magento/magento2/pull/25664/957e2fd64b6db4bf86baf86fed8fecde/Statics/allure-report-b2b/index.html#suites/51d3ab4d5d77a0bc6a7ea344e9e7e6be/44e51422a8c243df/

**Review by lenaorobei — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @lenaorobei, thank you for the review.
[ENGCOM-7145](https://jira.corp.magento.com/browse/ENGCOM-7145) has been created to process this Pull Request
:eight_spoked_asterisk: @lenaorobei, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**